stats.py

```diff
@@ -590,8 +629,17 @@ def handle_options(options):
     df = load_dataframe()
     langs = {x.lower(): x for x in df.columns}
     query = [x.lower() for x in options.search]
+    unstaged_solutions = []
+    unstaged_core_files = []
+    tbsolutions = []
```

Comment:
unstaged_solutions for new / modified solutions
unstaged_core_files for new/ modified core files ( stats.py, test, stats.exs ) 

unstaged_core_files the only utility I see for this is: stats.py changes would force stats.py --build to build all problems ignoring all other new/adds. I think decision making should be done in ./test script.

Reply:
This seems ok to me.